### PR TITLE
[O2-1012] Fix for python mismatch on macOS 10.15.2 when compiling ROO…

### PR DESCRIPTION
**Pull request by pzhristov**
…T. Bump ROOT to v6-20-02-alice3

### CODE REVIEW

File: root.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ COMPILER_CC=cc
 COMPILER_CXX=c++
 COMPILER_LD=c++
 case $PKGVERSION in
-  v6-18*) 
+  v6-20*)
```
Comment (ktf) [inline root.sh:45]: How about doing `v6-18*|v6-20*)` so that we can backport this, if needed?

Comment (pzhristov) [reply root.sh:45]: I think we should completely abandon v6-18.

Comment (ktf) [reply root.sh:45]: ok

**Comment by davidrohr** [comment]
@pzhristov : I had a look at the failing tests, but these are not normal ctests failing, this is ROOT failing to interprete / compile macros.
This was working before, so this might be a regression in 6.20?
I would not simply disable them, but I'd try to understand what code fails to compile in 6.20 that worked with 6.18. Perhaps this needs a fix in ROOT.

**Comment by pzhristov** [comment]
@davidrohr I made a pool request concerning the 4 failing O2 tests (out of 510), see 
 https://github.com/AliceO2Group/AliceO2/pull/3299
In the three cases the proble was in the O2 macro. What concerns macros/run_trac_its.C, there we include some files and end up using a huge json.h, where a template instantiation in swap doesn't work. I switched temporarily this test off.

**Comment by davidrohr** [comment]
@pzhristov : I see, but I would rather try to fix this, perhaps getting rid of the json.h and moving it to a cxx. Because if the macro doesn't work with the new ROOT, it also means we cannot run it, which might affect ITS reconstruction.

**Comment by pzhristov** [comment]
@davidrohr I propose to merge the changes and then work on the fix since it may take some time.

**Comment by davidrohr** [comment]
ok, you are not breaking my code :) But who is working to fix it?

**Comment by pzhristov** [comment]
I will try myself and contact the original authors in case I do not find quickly any solution.

**Pull request merged**